release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+:_content-type: ASSEMBLY
+[id="release-notes-40"]
+= Red Hat Advanced Cluster Security for Kubernetes 4.0
+include::modules/common-attributes.adoc[]
+:context: release-notes-40
+
+toc::[]
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime stages of the application lifecycle. It deploys in your infrastructure and integrates with your DevOps tools and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+|{product-title-short} version |Released on
+|`4.0` |1 May 2023
+|====
+
+[id="about-this-release-40"]
+== About this release
+
+{product-title-short} 4.0 includes the following new features, improvements, and updates:
+
+* <<major-release-version-change>>
+* <<external-postgresql-database-rn40>>
+* <<acs-cloud-release-rn40>>
+* <<telemetry-rn40>>
+* <<rhcos-node-scan-rn40>>
+* <<process-listening-endpoints-rn40>>
+* <<network-graph-rn40>>
+* <<fips-compliance-rn40>>
+* <<token-expiry-alert-rn40>>
+* <<sensor-resync-rn40>>
```

Comment:
I think this would be outside of the scope of this PR (because it looks like this style of within-the-same-document linking has been in use in the RHACS rel notes for a while now), but as a follow-up, it might be worth double-checking on whether this type of linking has any unforeseen issues, particularly as it relates to the Portal (it does work fine there now, in my quick checking) and any upcoming tooling. Maybe check with a DPM on this later, since it doesn't adhere to current guidelines, but might indeed be OK. I just don't know how much it has been tested.

Reply:
Yeah, I think Andrea recommended linking to the sections for the last release notes, and I did make sure afterwards that it didn't break the Customer Portal, but I will check with Kathryn to make sure there aren't any issues.